client/pages/Login/LoginForm/LoginForm.tsx

```diff
@@ -38,10 +38,13 @@ const LoginForm: React.FC<Props> = props => {
   }
 
   const onLogin = () => {
-    const fieldErrors = LoginValidator.localValidate(email, password)
+    let fieldErrors = LoginValidator.localValidate(email, password)
+    if (invitedUser && invitedUser.status !== 'active') {
+      fieldErrors = LoginValidator.invitationValidate(email, password, password2)
+    }
     setErrors(fieldErrors)
 
-    if (!fieldErrors.password && !fieldErrors.email) {
```

Comment:
we could add a field : `fieldErrors.error: boolean` if we have error?
We could use it in the if clause `fieldErrors.error`
And set it in `LoginValidator.localValidate`